docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/index.rst

```diff
@@ -35,9 +34,34 @@ GBD 2019 Modeling Strategy
 Multiple myeloma in GBD 2019
 ++++++++++++++++++++++++++
 
-.. todo::
+In GBD 2019, MM includes death and disability resulting from malignant neoplasms of plasma cells, including ICD-10 codes such as C90.0. The GBD modelling strategy can be found in the GBD YLD Capstone Appendix [GBD-2019-YLD-Capstone-Appendix-1-Neoplasms]_. 
+
+There is nothing custom about how MM is modeled in GBD compared to the other cancer causes that go through our standard fatal and non-fatal modeling pipelines.
 
-   Describe Multiple myeloma in GBD 2019.
+.. list-table:: 
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - sequelae
+     - health states
+     - health state lay descriptions
```

Comment:
Are durations provided here? They were for lung cancer and his could be relevant to list here!